bin/pem-python-remove

```diff
@@ -2,7 +2,14 @@
 [ "$PEM_DEBUG" = "yes" ] && set -x
 
 pem-default() {
+    [ "$1" = "-h" ] && usage && exit 0
```

Comment:
@ivan-podorozhnyi don't do this. Please take a look for https://github.com/extsoft/pem/commit/90f87066d1a54814ea5134bd37429baa96555b83